pom.xml

```diff
@@ -149,6 +149,12 @@
             <version>4.1.18.Final</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
In https://netty.io/wiki/forked-tomcat-native.html it has a `classifier` field. Why do you think it can be missing here?

Reply:
Refer this as the full instruction. https://netty.io/wiki/forked-tomcat-native.html#how-to-download-netty-tcnative-boringssl-static